sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -1386,6 +1388,12 @@ struct _pi_kernel : _pi_object {
     return true;
   }
 
+  void retain() {
```

Comment:
Please add a comment explaining that the caller must lock access to the kernel and program.

Reply:
Added comment.